.dockerignore

```diff
@@ -1,16 +1,9 @@
 **/__pycache__/
 node_modules/
-.github/
-.pytest_cache/
 static/
-terraform/
 .coverage
 *.db
 *.egg-info
-.flake8
-.pre-commit-config.yaml
-compose.yml
 *.log
 *.mo
 .DS_Store
-.env
```

Comment:
this one isn't tracked in version control, correct?

Reply:
If we want to be sure, we can also add `.pytest_cache/` to our `.gitignore`